localgov_demo.module

```diff
@@ -20,6 +21,15 @@ function localgov_demo_modules_installed($modules, $is_syncing) {
     in_array('localgov_demo', $modules)
   ) {
 
+    // Set all alert banners to be live by default.
+    $alert_banner_ids = \Drupal::entityQuery('localgov_alert_banner')->execute();
+    $alert_banner_entities = \Drupal::entityTypeManager()->getStorage('localgov_alert_banner')->loadMultiple($alert_banner_ids);
+
+    foreach ($alert_banner_entities as $alert_banner) {
+      $alert_banner->set('moderation_state', 'published');
```

Comment:
This line looks to be causing an error when enabling the module:

```
In ContentEntityBase.php line 587:
                                      
  [InvalidArgumentException]          
  Field moderation_state is unknown.  
                                      

Exception trace:
  at /app/web/core/lib/Drupal/Core/Entity/ContentEntityBase.php:587
 Drupal\Core\Entity\ContentEntityBase->getTranslatedField() at /app/web/core/lib/Drupal/Core/Entity/ContentEntityBase.php:568
 Drupal\Core\Entity\ContentEntityBase->get() at /app/web/core/lib/Drupal/Core/Entity/ContentEntityBase.php:628
 Drupal\Core\Entity\ContentEntityBase->set() at /app/web/modules/contrib/localgov_demo/localgov_demo.module:29
 localgov_demo_modules_installed() at n/a:n/a
 call_user_func_array() at /app/web/core/lib/Drupal/Core/Extension/ModuleHandler.php:403
 Drupal\Core\Extension\ModuleHandler->invokeAll() at /app/web/core/lib/Drupal/Core/Extension/ModuleInstaller.php:367
 Drupal\Core\Extension\ModuleInstaller->install() at /app/web/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php:83
 Drupal\Core\ProxyClass\Extension\ModuleInstaller->install() at /app/vendor/drush/drush/src/Drupal/Commands/pm/PmCommands.php:103
 Drush\Drupal\Commands\pm\PmCommands->enable() at n/a:n/a
 call_user_func_array() at /app/vendor/consolidation/annotated-command/src/CommandProcessor.php:257
 Consolidation\AnnotatedCommand\CommandProcessor->runCommandCallback() at /app/vendor/consolidation/annotated-command/src/CommandProcessor.php:212
 Consolidation\AnnotatedCommand\CommandProcessor->validateRunAndAlter() at /app/vendor/consolidation/annotated-command/src/CommandProcessor.php:176
 Consolidation\AnnotatedCommand\CommandProcessor->process() at /app/vendor/consolidation/annotated-command/src/AnnotatedCommand.php:313
 Consolidation\AnnotatedCommand\AnnotatedCommand->execute() at /app/vendor/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at /app/vendor/symfony/console/Application.php:1027
 Symfony\Component\Console\Application->doRunCommand() at /app/vendor/symfony/console/Application.php:273
 Symfony\Component\Console\Application->doRun() at /app/vendor/symfony/console/Application.php:149
 Symfony\Component\Console\Application->run() at /app/vendor/drush/drush/src/Runtime/Runtime.php:118
 Drush\Runtime\Runtime->doRun() at /app/vendor/drush/drush/src/Runtime/Runtime.php:48
 Drush\Runtime\Runtime->run() at /app/vendor/drush/drush/drush.php:72
 require() at /app/vendor/drush/drush/drush:4
```

Reply:
This looks to be due to the profile not including Alert Banner 1.2.0, so not a problem with this module. We either need to do a new release of the profile or change the version requirements as suggested by @Adnan-cds here: https://github.com/localgovdrupal/localgov/issues/365

Reply:
Thanks @stephen-cox , this should be resolved with https://github.com/localgovdrupal/localgov/releases/tag/2.1.7

Reply:
Does that mean we can resolve this conversation @stephen-cox  ( not sure on protocol! )